data/json/items/vehicle/motors.json

```diff
@@ -76,5 +76,16 @@
     "category": "veh_parts",
     "price": 2000,
     "price_postapoc": 100
+  },
+  {
+    "type": "GENERIC",
+    "id": "motor_train1300",
+    "name": { "str": "1300hp electric train engine" },
+    "description": "A 1300hp 3-phase 60 Hz electric engine.  Normally used in trains.",
```

Comment:
I assume it would work out of normal batteries and wouldn't require power inverter to work. That description could be quite misleading and confusing for end user.

Reply:
What description would you suggest (I basically just copied buzz words from wikipedia)

Reply:
Removing the "3-phase 60 Hz" part would remove the confusion. Could add some new adjectives in their place to flavor the text.

Reply:
How about "extended-use"?

Reply:
that might be too generic; discord already gave me ideas to make more train engines; I just did this one first, because it was the most powerful and I wanted some power for my game :D

Reply:
but since I don't have any better ideas 1300hp extended-use electric engine should do it